utils/update_dump.py

```diff
@@ -25,7 +24,7 @@ def error(self, *args):
     self.has_error = True
 
   def generate_dump(self, source, dump_dir):
-    cmd = shlex.split(self.compiler) + ["--dump-path=%s" % dump_dir, source]
+    cmd = [self.compiler, "--dump-path=%s" % dump_dir, source]
```

Comment:
`--dump=path` was removed?

Reply:
Yes, and `update_dump.py` is no longer being used to regenerate the test files for the parser tests. It should no longer be touched by the PR

Reply:
this just reverts the source to where it was prior to my changes

Reply:
of course! I only saw the commit, not the final state. :)